src/FeatureFlagHelper.php

```diff
@@ -32,9 +32,7 @@ public function registerFeatureFlags()
                 $features = [];
             }
 
-            $world = new World();
-
-            \Feature\Feature::create($world, $features);
+            Feature::create($features);
```

Comment:
Take advantage of Laravel facades here even dynamic facades it will make it for you https://laravel.com/docs/5.7/facades#real-time-facades